snap/info.go

```diff
@@ -65,3 +71,13 @@ type AppInfo struct {
 	Plugs   map[string]*PlugInfo
 	Slots   map[string]*SlotInfo
 }
+
+// StoreInfo provides specific information for a store hosted snap.
+type StoreInfo struct {
+	LastUpdated     time.Time
+	DownloadSha512  string
+	DownloadSize    int64
+	AnonDownloadURL string
+	DownloadURL     string
```

Comment:
I don't want to make a decision based on the token inside infoFromRemote because it's unclear given the stage engine properties how long it will take to go from there to actually running download, I fear we need to keep the two urls around for a while (until we can always auth (post 16.04) or the store fixes itself to return empty anon when the snap is private)